src/pages/difficulty.js

```diff
@@ -0,0 +1,93 @@
+import * as React from 'react';
+import { Link } from 'gatsby';
+import '../styles/style.scss';
+
+// data
+const data = [
+    {
+        text: 'Very easy',
+        dc:'5'
+    },
+    {
+        text: 'Easy',
+        dc:'10'
+    },
+    {
+        text: 'Moderate',
+        dc:'15'
+    },
+    {
+        text: 'Hard',
+        dc:'20'
+    },
+    {
+        text: 'Very hard',
+        dc:'25'
+    },
+    {
+        text: 'Nearly impossible',
+        dc:'30'
+    }
+];
+
+// data
+const track = [
+    {
+        text: 'Soft surface such as snow',
+        dc:'10'
+    },
+    {
+        text: 'Dirt or grass',
+        dc:'15'
+    },
+    {
+        text: 'Bare stone',
+        dc:'20'
+    },
+    {
+        text: 'Each day since the creature passed',
+        dc:'(+5)'
+    },
+    {
+        text: 'The creature left a trail, such as blood',
+        dc:'(-5)'
+    }
```

Comment:
Are these values only ever going to be displayed? Long term, is the plan to do math with them? Could be smart to have a `displayDc` attribute for printing and a `dcValue` attribute to do math with.

Reply:
My thought was just display. The numbers are only meant to be representative of static DCs and/or modifiers. I think I might have confused things by wrapping those in parentheses. It was for my own visibility in testing and I forgot to remove them.

Thought it could appear something like this:
![image](https://user-images.githubusercontent.com/10962974/132112852-b60ef997-0edd-43a6-8a7a-60e270e16e03.png)
 
Does that make sense?

Reply:
Ahh, yeah. if it's just for display purposes it's fine as is